website/docs/components/form/super-select/partials/code/component-api.md

```diff
@@ -13,7 +13,12 @@ While the [ember-power-select documentation](https://ember-power-select.com/docs
 
 !!! Info
 
-The default values of some [ember-power-select](https://ember-power-select.com/docs/api-reference) options have been overridden in some cases to either resolve accessibility issues or to customize according to our needs. These options are documented below.
+The default values of some [ember-power-select](https://ember-power-select.com/docs/api-reference) options have been overridden to either resolve accessibility issues or to customize according to our needs.
+
+Overrides to fix accessibility issues:
+
+- `renderInPlace`: Always set as `true`. (The opener element or “trigger” should be next to dropdown content it hides & reveals.)
+- `searchFieldPosition`: Always set to `"before-options"`. (Rendering the search input within the trigger is not compliant because it nests interactive elements within each other.)
```

Comment:
```suggestion
- `searchFieldPosition`: Always set to `"before-options"`. Rendering the search input within the trigger is not compliant because it nests interactive elements within each other.
```